test/e2e/network/ingress.go

```diff
@@ -740,10 +740,11 @@ var _ = SIGDescribe("Loadbalancing: L7", func() {
 			}
 			if jig.Ingress == nil {
 				ginkgo.By("No ingress created, no cleanup necessary")
-				return
+			} else {
+				ginkgo.By("Deleting ingress")
+				jig.TryDeleteIngress()
 			}
-			ginkgo.By("Deleting ingress")
-			jig.TryDeleteIngress()
+			nginxController.TearDown()
```

Comment:
Instead of introducing this else above, can we add "defer nginxController.TearDown" before this if?

Reply:
Done.